pkg/agent/flowexporter/connections/connections.go

```diff
@@ -37,8 +38,20 @@ var serviceProtocolMap = map[uint8]corev1.Protocol{
 	132: corev1.ProtocolSCTP,
 }
 
-type ConnectionStore struct {
+type ConnectionStore interface {
+	// Run enables the periodical polling of conntrack connections at a given flowPollInterval.
+	Run(stopCh <-chan struct{})
+	// GetConnByKey gets the connection in connection map given the connection key.
+	GetConnByKey(flowTuple flowexporter.ConnectionKey) (*flowexporter.Connection, bool)
+	// ForAllConnectionsDo execute the callback for each connection in connection map.
+	ForAllConnectionsDo(callback flowexporter.ConnectionMapCallBack) error
+	// DeleteConnectionByKey deletes the connection in connection map given the connection key.
+	DeleteConnectionByKey(connKey flowexporter.ConnectionKey) error
```

Comment:
I think that from a design perspective, it is a bit surprising that the FlowExporter is in charge of deleting connections from the ConnectionStore. I understand why: we need to make sure the connection information is preserved until the last record can be sent. But maybe it is worth a comment.

Reply:
I don't see the comment?

Reply:
Added the comment in flow exporter code. Moved it here.